docs/concepts/browsers-used-by-office-web-add-ins.md

```diff
@@ -16,35 +16,37 @@ Which browser is used depends on:
 
 The following table shows which browser is used for the various platforms and operating systems.
 
-|**OS / Platform**|**Browser**|
+|**OS**|**Office version**|**Edge WebView2 (Chromium-based) installed**|**Browser**|
 |:-----|:-----|:-----|:-----|:-----|:-----|:-----|
-|Office on the web|The browser in which Office is opened.|
-|Mac|Safari|
-|iOS|Safari|
-|Android|Chrome|
-|Windows / non-subscription Office 2013 or later|Internet Explorer 11|
-|Windows 10 ver. < 1903 / Office 365|Internet Explorer 11|
-|Windows 10 ver. >= 1903 / Office 365 ver < 16.0.11629<sup>1</sup>|Internet Explorer 11|
-|Windows 10 ver. >= 1903 / Office 365 ver >= 16.0.11629<sup>1</sup>|Microsoft Edge<sup>2, 3</sup>|
+|any|Office on the web|N/A|The browser in which Office is opened.|
+|Mac|any|N/A|Safari|
+|iOS|any|N/A|Safari|
+|Android|any|N/A|Chrome|
```

Comment:
Should spell out N/A (see [style guide](https://docs.microsoft.com/en-us/style-guide/scannable-content/tables#content))
```suggestion
|any|Office on the web|Not applicable|The browser in which Office is opened.|
|Mac|any|Not applicable|Safari|
|iOS|any|Not applicable|Safari|
|Android|any|Not applicable|Chrome|
```